usagov/vote-gov-drupal
# Vote-3148: Cypress Test Updates

## Jira ticket

[Vote-3148](https://cm-jira.usa.gov/browse/VOTE-31480)

## Description

This Pr will add more tests for the frontend.
- Test to verify that the touch points module is opening as expected 
- Adding test to the pipeline to check for the redirect issue.  This will be ran in the pipeline on prod mon-fri at 9am and will flag if the issue is present 

## Deployment and testing

### Post-deploy steps

n/a

### QA/Testing instructions

1. cd into `testing` and run `npm run cy:frontEnd` and verify that all tests are passing
2. [check out the test run](https://app.circleci.com/pipelines/github/usagov/vote-gov-drupal/102113/workflows/9acd7fac-a1b6-4b80-a42a-202914dbdf1b/jobs/110412) and look at the test and how it will output errors 

## Checklist for the Developer




## Checklist for the Peer Reviewers
